core/arch/arm/plat-hikey/conf.mk

```diff
@@ -1,7 +1,7 @@
 include core/arch/$(ARCH)/plat-$(PLATFORM)/platform_flags.mk
 
 ifeq ($(CFG_ARM64_core),y)
-CFG_WITH_LPAE := y
+CFG_WITH_LPAE ?= y
```

Comment:
Indeed. Basically, we want `override CFG_WITH_LPAE := y`, but it would be nice if we could print a message in case someone tries to define the variable externally. I'll see if I can write a convenient macro.